languages/python/exercises/concept/chaitanas-colossal-coaster/.docs/introduction.md

```diff
@@ -95,7 +81,7 @@ ex_list.pop()
 #=> [2]
```

Comment:
To avoid confusion:
ex_list.pop()
#=> 3
print(ex_list)
#=> [2]

Reply:
@valentin-p  -- While this may be more readable, it's not the convention for Python code examples.  We really should be modeling here what the REPL prints out, so that the student has a sense of exactly what they will be seeing.  This holds true for examples in the Python docs themselves:

![image](https://user-images.githubusercontent.com/5923094/100474066-a7984780-3094-11eb-81e5-f6bbc9282b5a.png)

And is also used widely in books and posts about Python.  I am still working on Pythons contributing guide, and also let some inconsistent examples slip through -- but Python code for our docs should conform to the Python docs and to the syntax of [docttest](https://docs.python.org/3/library/doctest.html).

Reply:
See new #2868